tajo-client/src/main/java/org/apache/tajo/client/v2/LegacyClientDelegate.java

```diff
@@ -72,10 +72,20 @@ public int executeUpdate(String sql) throws TajoException {
   }
 
   @Override
-  public ResultSet executeSQL(String sql) throws TajoException {
+  public ResultSet executeSQL(String sql) throws TajoException, QueryFailedException, QueryKilledException {
```

Comment:
It means a documentation. This exceptions will let users to know what exception should be handled.